syntax = "proto3";

package communicator_objects;

message ActionSpecProto {
    int32 num_continuous_actions = 1;
    int32 num_discrete_actions = 2;
    repeated int32 discrete_branch_sizes = 3;
    repeated string action_descriptions = 4;
}

message BrainParametersProto {
    string brain_name = 1;
    bool is_training = 2;
    ActionSpecProto action_spec = 3;
}
